README.windows.md

```diff
@@ -242,6 +242,14 @@ If you are building for 64-bit windows, the steps are essentially the same. Just
 
 - run gdb under the windows console (cmd) instead. gdb [does not function properly](https://www.cygwin.com/ml/cygwin/2009-02/msg00531.html) under mintty with non-cygwin applications.
 
+### GDB not attaching to the right process
+
+- Use the PID from the windows task manager instead of unix style command line tools. You may need to add the PID column if it is not shown by default.
```

Comment:
i don't know what all of the columns mean, but WINPID is the one you want for `gdb -p`
```
$ ps
      PID    PPID    PGID     WINPID   TTY     UID    STIME COMMAND
     1096     392    1096       3884  pty0    1000 22:19:04 /usr/bin/vim
```